component.go

```diff
@@ -70,3 +70,18 @@ func (cs ComponentService) GetAll(ctx context.Context, projectUUID uuid.UUID, po
 	p.TotalCount = res.TotalCount
 	return
 }
+
+func (cs ComponentService) Create(ctx context.Context, projectUUID string, component Component) (c Component, err error) {
+	if err != nil {
+		return
+	}
```

Comment:
This will always be `false`, any reason why it's here?

Reply:
main reason is i made a mistake, fixed, thanks for catching this